Makefile

```diff
@@ -468,7 +468,7 @@ install.completions:
 
 install.systemd:
 	install ${SELINUXOPT} -D -m 644 contrib/systemd/crio.service $(PREFIX)/lib/systemd/system/crio.service
-	ln -sf crio.service $(PREFIX)/lib/systemd/system/cri-o.service
+	ln -sf $(PREFIX)/lib/systemd/system/crio.service $(PREFIX)/lib/systemd/system/cri-o.service
```

Comment:
@haircommander Perhaps requesting `ln` create a relative link might work?  At a guess, the target path (including BUILDROOT reference) is being expanded before the link is written to the filesystem, if that makes sense -- and it succeeds despite no such target existing, due to the 'force' flag.

Reply:
thanks for the suggestion! I looked into a bit more, and it turns out symlinks between absolute paths in rpms are just not allowed. I am looking into a new solution now, which may not involve these changes...

Reply:
No probs!  Thanks for investigating further